hawkbit-repository/src/main/java/org/eclipse/hawkbit/repository/TargetManagement.java

```diff
@@ -205,7 +205,7 @@ public Long countTargetsAll() {
     @PreAuthorize(SpringEvalExpressions.HAS_AUTH_READ_TARGET)
     public Slice<Target> findTargetsAll(@NotNull final TargetFilterQuery targetFilterQuery,
             @NotNull final Pageable pageable) {
-        return findTargetsAll(RSQLUtility.parse(targetFilterQuery.getQuery(), TargetFields.class), pageable);
+        return findTargetsAll(RSQLUtility.parse(targetFilterQuery.getQuery().toLowerCase(), TargetFields.class), pageable);
```

Comment:
I would prefer to put the target filter string  `toLowerCase()` to the `RSQLUtility`, then we don't need to call the `toLowerCase()` in each method, because I think all queries by filter string are case-insensitive isn't it?